content/en/blog/2023-05-01-april-update/index.md

```diff
@@ -0,0 +1,221 @@
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
+#### Terraform-controller
+
+#### Flux Subsystem for Argo
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
+It's important to keep you up to date with new features and developments
+in Flux and provide simple ways to see our work in action and chat with
+our engineers.
+
+### Recent Events (ICYMI) 📺
+
+We feel blessed to have such a big community of users, contributors and
+integrators and so many are happy to talk about their experiences. In
+April here are a couple of talks we would like to highlight.
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+May - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+#### Flux Bug Scrub
+
+Our Flux Bug Scrubs still are happening on a weekly basis and remain one
+of the best ways to get involved in Flux. They are a friendly and
+welcoming way to learn more about contributing and how Flux is organised
+as a project.
+
+The next dates are going to be:
+
+- [2023-xx-yy 12:00 UTC, 14:00 CEST](/#calendar)
+- [2023-xx-yy 17:00 UTC, 19:00 CEST](/#calendar)
+- [2023-xx-yy 12:00 UTC, 14:00 CEST](/#calendar)
+- [2023-xx-yy 17:00 UTC, 19:00 CEST](/#calendar)
+
+We are flexible with subjects and often go with the interests of the
+group or of the presenter. If you want to come and join us in either
+capacity, just show up or if you have questions, reach out to Kingdon on
+Slack.
+
+We really enjoyed this [demo of the k3d git
+server](https://www.youtube.com/watch?v=hNt3v0kk6ec)
+recently. It's a local Git server that runs outside of Kubernetes, to
+support offline dev in a realistic but also simple way that does not
+depend on GitHub or other hosted services.
+
+## In other news
+
+### Your Community Team
+
+### People writing/talking about Flux
+
+We love it when you all write about Flux and share your experience,
+write how-tos on integrating Flux with other pieces of software or other
+things. Give us a shout-out and we will link it from this section! ✍
+
+- <https://grafana-operator.github.io/grafana-operator/blog/2023/03/29/install-grafana-operator-using-flux-and-kustomize/>
```

Comment:
For some additional context around the grafana-operator link.
How about something like this?

The grafana-operator team have recently started to ship their Kustomize manifests using OCI with the help of Flux artifact. As a part of this, they have written a small blog on how to install grafana-operator using Flux and how to manage grafana dashboards as code.